replace_unix.go

```diff
@@ -6,21 +6,24 @@ import (
 	"syscall"
 )
 
+func mprotectCrossPage(addr uintptr, len int, prot int) {
+	pageSize := syscall.Getpagesize()
+	for p := pageStart(addr); p <= addr + uintptr(len); p += uintptr(pageSize) {
+		page := rawMemoryAccess(p, syscall.Getpagesize())
```

Comment:
you can use `pageSize` here instead of calling `Getpagesize`

Reply:
ok, 3q